### PR TITLE
Issue37 sp deck remove function fix

### PR DESCRIPTION
**Pull request by terri-hewitt**
Split remove function into two methods (one for the English deck and one for the Spanish deck).

### CODE REVIEW

**Comment by Lucas-Campos-Davis** [comment]
Did you fix it so it can remove cards even if there is no joker on the board?

**Comment by terri-hewitt** [comment]
Yes, it will remove cards even if no joker is on the board.

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -41,40 +41,63 @@ public void dealFour() {
...
-                                        break;  //Found a card that is the same suit and higher value than removable card
-                                    }
+                    int j;
+                    if (checkCard.value == 13) {   //Check if the checkCard is a Joker
+                        for (j = i; j < 4; j++) { //Start where "i" left off
+                            checkCard = cols.get(j).getTopCard();   //Check rest of columns for a card that's not a Joker to remove the selected card
+                            if ((checkCard.value != 13) && (removableCard.value < checkCard.value)) { //Check the card is not a Joker and the value is > checkCard
+                                if (Objects.equals(removableCard.suit, checkCard.suit)) { //Check that the suits match
+                                    cols.get(columnNumber).removeTopCard(); //Remove the selected card
+                                    JOptionPane.showMessageDialog(null, "Other card found", "You Win!! ", JOptionPane.INFORMATION_MESSAGE);
```
Comment (cairnsr) [inline src/main/java/models/Game.java]: Just a little confused here, do we need the "Other card found you win message"?

Comment (terri-hewitt) [reply src/main/java/models/Game.java]: Oh whoops! I was using that message for testing. I'll delete that line.

**Comment by cairnsr** [comment]
Alrighty looks great to me, Ill go ahead and merge

**Pull request merged**